### DISCUSSION TITLE
How can I keep the spacing between brackets for arguments?

### DISCUSSION THREAD

**Discussion by darthf1** [VincentLanglet/Twig-CS-Fixer#342] [Q&A] [answered]
Hi!

I'm trying out twig-cs-fixer and like a lot of the formatting that is being done!

Considering the diff below; how can I keep the spaces between brackets for arguments, so the diff is not like this:

```diff
- {% trans with { 'engagementName': engagementName } %}
+ {% trans with {engagementName: engagementName} %}
```

But like this:

```diff
- {% trans with { 'engagementName': engagementName } %}
+ {% trans with { engagementName: engagementName } %}
```

I tried the twig-cs-fixer ruleset, and disabling the `NamedArgumentSpacingRule`, but that didnt work for me.

```php
$ruleset
    ->addStandard(new TwigCsFixer())
    ->removeRule(NamedArgumentSpacingRule::class)
;
```

**Comment by VincentLanglet** [comment, marked as answer]
You should look at the PunctuationSpacingRule
https://github.com/VincentLanglet/Twig-CS-Fixer/blob/main/src/Rules/Punctuation/PunctuationSpacingRule.php

You can even configure it with `['}' => 1], ['{' => 1]`
